### ISSUE TITLE
Can't scope to nodes under Dependency node

### ISSUE THREAD

**Issue by davkean** [dotnet/project-system#1161]
You can scope to all nodes, except the NuGet/Projects/etc nodes:

![image](https://cloud.githubusercontent.com/assets/1103906/21923338/998deec4-d9c7-11e6-8fd5-fa837cb8cdcb.png)

For example:

![image](https://cloud.githubusercontent.com/assets/1103906/21923353/ace6db8e-d9c7-11e6-9316-58118f4d6479.png)

**Comment by srivatsn** [comment]
Dupe of #811?

**Comment by davkean** [comment]
yeah, sigh, just finding the same bugs again...

**Comment by abpiskunov** [comment]
Its not same bug. Scope command depends on guid and flag associated with item and now we switched to cps References which have different guid and flags. Other nodes dont have Reference flag and scope shows up for them. This logic lives in Cps and i had to choose what problem to solve, scope had lower priority. In order to fix it , we either need to bring command handlers and mayby UIHierarchy logic to core project system or change cps logic to more configurable.

Old issue that you opened did nothing when you clicked on Scope - that was fixed. Now scope command is not showing up.

Thanks,
Anton

On Jan 12, 2017, at 10:41 PM, David Kean <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #1161<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdotnet%2Froslyn-project-system%2Fissues%2F1161&data=02%7C01%7Canton.piskunov%40microsoft.com%7C61bb47e3a84e43da89a208d43b8fda55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636198936407880022&sdata=7GbRvRnm7j5pmE%2B5TMLfYCszjGfGdSsWn1P4JOTEInI%3D&reserved=0>.

-
